index.pug

```diff
@@ -0,0 +1,591 @@
+          span Explore &gt;
+      .sec5box1
+        a(href='https://razorpay.com/payment-buttons/' target='_blank')
+          svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#ffffff')
+            path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+          img.lazy.boxicon(data-src="/assets/education/paymentbutton.svg", alt="paymentbutton")
+          h2 Payment Buttons
+          ul
+            li Collect payments on your website with zero coding
+            li Accept one time &amp; recurring payments
+            li Go live in less than 5 mins
+          span Know More &gt;
+      .sec5box1
+        a(href='https://razorpay.com/payment-links/' target='_blank')
+          svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#ffffff')
+            path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+          img.lazy.boxicon(data-src="/assets/education/paymentlinksdark.svg", alt="paymentlinksdark")
+          h2 Payment Links
+          ul
+            li Create a link from dashboard or app
+            li Share via WhatsApp, SMS or social channels
+            li Set automated reminders and link expiry
+          span Explore &gt;
+      .sec5box1
+        a(href='https://razorpay.com/payment-pages/' target='_blank')
+          svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#ffffff')
+            path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+          img.lazy.boxicon(data-src="/assets/education/paymentpagesdark.svg", alt="paymentpagesdark")
+          h2 Payment Pages
+          ul
+            li Build branded web pages
+            li Zero coding involved
+            li Customise content, images, and links
+          span Explore &gt;
+  // section6 start
+  #OntheGoBusiness.section6.container1
+    img.onthegobg(src='images/Onthegobg.svg' alt='on-the-go-bg')
+    img(src='images/paymentsolutions.svg' alt='manage-your-business')
+    h1 Manage your business from anywhere, anytime
+    p Set up online presence, accept payments and track transactions on the go
+    ul.sec6pointers
+      li No coding required
+      li No integration
+      li Get started in minutes
+    .section6insideTop
+      .sec6box1Top
+        a(href='https://razorpay.com/payments-app/' target='_blank')
+          svg.svg-corner.svg-desktop.mobile(xmlns='http://www.w3.org/2000/svg' width='700' height='500' fill='#ffffff')
```

Comment:
can we have the SVG on the external files, instead of embedding it here, to it for all the sections